spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/SpatialDilatedConvolution.scala

```diff
@@ -147,6 +148,15 @@ class SpatialDilatedConvolution[T: ClassTag](
     }
   }
 
+  override def computeOutputShape(inputShape: Shape): Shape = {
+    val input = inputShape.toSingle().toArray
```

Comment:
add inputShape checking here.

Reply:
Do we need checking? As calling `shapeCheck` need a tensor not shape

Reply:
Updated.